tests/validation/cattlevalidationtest/core/test_machine_amazon.py

```diff
@@ -0,0 +1,552 @@
+import boto.ec2
+import logging
+import traceback
+
+from common_fixtures import *  # NOQA
+
+DEFAULT_TIMEOUT = 450
+access_key = os.environ.get('AMAZON_ACCESSKEY')
+secret_key = os.environ.get('AMAZON_SECRETKEY')
+
+# Amazon settings used for instance deployment
+vpc_id = "vpc-eb2d928e"
+subnet_id = "subnet-5775c020"
+security_group = "test"
+zone = "a"
+region = "us-west-2"
+ami = "ami-23ebb513"
+size = "32"
+instance_type = "t1.micro"
```

Comment:
I believe t1.micro is the old deprecated ec2 instance types.  Let's pick one that is more modern like t2.small for the "catch-all" tests.